src/main/java/net/minecraftforge/oredict/OreDictionary.java

```diff
@@ -126,33 +146,107 @@ public static void initVanillaEntries()
             registerOre("record",      Items.record_wait);
+
+            // blocks
+            registerOre("dirt",        Blocks.dirt);
+            registerOre("grass",       Blocks.grass);
+            registerOre("stone",       Blocks.stone);
+            registerOre("cobblestone", Blocks.cobblestone);
+            registerOre("gravel",      Blocks.gravel);
+            registerOre("sand",        new ItemStack(Blocks.sand, 1, WILDCARD_VALUE));
+            registerOre("sandstone",   new ItemStack(Blocks.sandstone, 1, WILDCARD_VALUE));
+            registerOre("sandstone",   new ItemStack(Blocks.red_sandstone, 1, WILDCARD_VALUE));
+            registerOre("netherrack",  Blocks.netherrack);
+            registerOre("obsidian",    Blocks.obsidian);
+            registerOre("glowstone",   Blocks.glowstone);
+            registerOre("endstone",    Blocks.end_stone);
+            registerOre("torch",       Blocks.torch);
+            registerOre("blockSlime",    Blocks.slime_block);
+            registerOre("blockPrismarine",
+                    new ItemStack(Blocks.prismarine, 1, BlockPrismarine.EnumType.ROUGH.getMetadata()));
+            registerOre("blockPrismarineBrick",
+                    new ItemStack(Blocks.prismarine, 1, BlockPrismarine.EnumType.BRICKS.getMetadata()));
+            registerOre("blockDarkPrismarine",
+                    new ItemStack(Blocks.prismarine, 1, BlockPrismarine.EnumType.DARK.getMetadata()));
+            registerOre("stoneGranite",          new ItemStack(Blocks.stone, 1, 1));
+            registerOre("stoneGranitePolished",  new ItemStack(Blocks.stone, 1, 2));
+            registerOre("stoneDiorite",          new ItemStack(Blocks.stone, 1, 3));
+            registerOre("stoneDioritePolished",  new ItemStack(Blocks.stone, 1, 4));
+            registerOre("stoneAndesite",         new ItemStack(Blocks.stone, 1, 5));
+            registerOre("stoneAndesitePolished", new ItemStack(Blocks.stone, 1, 6));
+            registerOre("blockGlassColorless", Blocks.glass);
+            registerOre("blockGlass",    Blocks.glass);
+            registerOre("blockGlass",    new ItemStack(Blocks.stained_glass, 1, WILDCARD_VALUE));
+            //blockGlass{Color} is added below with dyes
+            registerOre("paneGlassColorless", Blocks.glass_pane);
+            registerOre("paneGlass",     Blocks.glass_pane);
+            registerOre("paneGlass",     new ItemStack(Blocks.stained_glass_pane, 1, WILDCARD_VALUE));
+            //paneGlass{Color} is added below with dyes
+
+            // chests
             registerOre("chest",       Blocks.chest);
             registerOre("chest",       Blocks.ender_chest);
             registerOre("chest",       Blocks.trapped_chest);
             registerOre("chestWood",   Blocks.chest);
             registerOre("chestEnder",  Blocks.ender_chest);
             registerOre("chestTrapped", Blocks.trapped_chest);
+            registerOre("workbench",   Blocks.crafting_table);
```

Comment:
Crafting table, is not a chest... Its also not in your inital post about what you're changing.